testkit/src/main/scala/org/bitcoins/testkit/core/gen/CreditingTxGen.scala

```diff
@@ -29,12 +29,15 @@ sealed abstract class CreditingTxGen {
 
   /** Generator for non-script hash based output */
   def nonSHOutput: Gen[BitcoinUTXOSpendingInfo] = {
-    Gen.oneOf(p2pkOutput,
-              p2pkhOutput,
-              multiSigOutput, /*cltvOutput,*/ csvOutput,
-              multiSignatureWithTimeoutOutput,
-              conditionalOutput,
-              p2wpkhOutput)
+    Gen.oneOf(
+      p2pkOutput,
+      p2pkhOutput,
+      /*p2pkWithTimeoutOutput,*/
```

Comment:
this has to be commented out as well since it uses the `nLockTime` field? Ugh, that is frustrating.

Reply:
yeah :/ I have some ideas for fixing this, will write up an issue